drivers/flash/flash_stm32_xspi.c

```diff
@@ -2468,9 +2467,17 @@ static struct flash_stm32_xspi_data flash_stm32_xspi_dev_data = {
 					: HAL_XSPI_CSSEL_NCS2),
 #endif
 			.FreeRunningClock = HAL_XSPI_FREERUNCLK_DISABLE,
-#if defined(OCTOSPI_DCR4_REFRESH)
+#if defined(OCTOSPI_DCR1_DLYBYP) || defined(XSPI_DCR1_DLYBYP)
+			.DelayBlockBypass = (DT_PROP(STM32_XSPI_NODE, dlyb_bypass)
+					? HAL_XSPI_DELAY_BLOCK_BYPASS
+					: HAL_XSPI_DELAY_BLOCK_ON),
+#endif /* xXSPI_DCR1_DLYBYP */
```

Comment:
`XSPI_DCR1_DLYBYP` and `DelayBlockBypass` don't seem to be available for H7RS.
But they are valid for STM32H5, so it can be decided to be kept in another commit

Reply:
put in commit 5a948ac12eb447e0117148a1cd35b41f65f2879e